samples/bookinfo/routing/bookinfo-gateway.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
doesn't this have to be used in some script to do the actual testing?

Reply:
yes, it has to, let me submit a separate PR for that.